samples/MvcSandbox/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "commands": {
-    "web": "Microsoft.AspNet.Hosting server=Microsoft.AspNet.Server.WebListener server.urls=http://localhost:5001",
+    "web": "Microsoft.AspNet.Server.Kestrel",
```

Comment:
Does this work without a `hosting.json` file? If not, please `git add` that file to this commit.